### ISSUE TITLE
Did anyone has trained the model on coco? What's the map?

### ISSUE THREAD

**Issue by ye-ye** [YunYang1994/tensorflow-yolov3#190]
Did anyone has trained the model on coco? What's the map?

**Comment by YunYang1994** [comment]
![41560154603_ pic](https://user-images.githubusercontent.com/30433053/59181996-2a74cc00-8b9b-11e9-8ce1-fc9dd1b0048e.jpg)

**Comment by ilyes-hm** [comment]
Could you share the parameters that lead to that result? (LR scheduler, optimizer, batch size...) please?

**Comment by RubenS02** [comment]
Hi @YunYang1994, I tried training on COCO but achieved only 0.45 mAP. Could you please share the hyperparameters you used? Thanks

**Comment by akira831218** [comment]
@YunYang1994  I need it for research ,too.   Thanks  a lot!

**Comment by Taruio** [comment]
@YunYang1994 How many epochs had you trained? Thank you!

**Comment by YunYang1994** [comment]
default parameters

**Comment by mumu0419** [comment]
> Hi @YunYang1994, I tried training on COCO but achieved only 0.45 mAP. Could you please share the hyperparameters you used? Thanks

I want to know how to train yolov3 on coco dataset?
For I am a beginner of object detection,I don't know how to transfer coco dataset to voc format.
